config/cime/config_compilers.xml

```diff
@@ -19,7 +19,21 @@
      </SLIBS>  
      <SCC MPILIB="mpi-serial">gcc</SCC>
      <SFC MPILIB="mpi-serial">gfortran</SFC>
-   </compiler>
+</compiler>
+   
+<compiler MACH="container">
+  <CPPDEFS>
+    <!-- these flags enable nano timers -->
+    <append MODEL="gptl"> -DHAVE_NANOTIME -DBIT64 -DHAVE_VPRINTF -DHAVE_BACKTRACE -DHAVE_SLASHPROC -DHAVE_COMM_F2C -DHAVE_TIMES -DHAVE_GETTIMEOFDAY </append>
+  </CPPDEFS>
+  <NETCDF_PATH>/usr/local</NETCDF_PATH>
+  <PNETCDF_PATH>/usr/local</PNETCDF_PATH>
+  <LDFLAGS>
+  </LDFLAGS>
+  <SLIBS>
+    <append> -lnetcdf -lnetcdff -llapack -lblas </append>
+  </SLIBS>
+</compiler>
```

Comment:
similar to https://github.com/NorESMhub/NorESM_LandSites_Platform/pull/33/commits/d1f3909d747bef614644fbb7a110a2b8478a0c08#r663801126